README.md

```diff
@@ -1,11 +1,99 @@
 # Load Secrets from 1Password - GitHub Action
 
-The action to load secrets from [1Password Connect](https://1password.com/secrets/) into GitHub Actions.
+This action loads secrets from [1Password Connect](https://1password.com/secrets/) into GitHub Actions.
+
 
 Specify right from your workflow YAML which secrets from 1Password should be loaded into your job, and the action will make them available as environment variables for the next steps.
 
+## Prerequisites 
+ - [1Password Connect](https://support.1password.com/secrets-automation/#step-2-deploy-a-1password-connect-server) deployed in your infrastructure
+
 ## Usage
 
+You can configure the action to use either 1Password connect instance or service account.
```

Comment:
Connect is capitalized, since it is the name of a product we've built.
```suggestion
You can configure the action to use either 1Password Connect instance or service account.
```

Reply:
fixed